sites/cheerp/src/pages/index.astro

```diff
@@ -1,61 +1,243 @@
+			<div class="flex justify-center items-center">
+				<Image src={DrawingSourcerer} alt="Sourcerer" class="w-64" />
+			</div>
+			<h1
+				class="text-6xl text-center bg-gradient-to-r from-cheerp to-[#8e88ff] text-transparent bg-clip-text"
+			>
+				We love open sourcerers
+			</h1>
+			<p class="text-center text-4xl text-stone-300 py-4">
+				Cheerp is licensed under Apache 2.0 License with LLVM exceptions. We're
+				grateful for your contributions.
+			</p>
+
+			<div class="flex justify-center items-center">
+				<LinkButton
+					type="primary-cheerp"
+					href="https://github.com/leaningtech/cheerp-meta"
+					label="Contribute"
+				/>
+			</div>
+			<div
+				class="grid grid-cols-3 py-8 text-stone-300 px-2 sm:px-8 md:px-16 lg:px-32 xl:-px-72 2xl:px-96"
+			>
+				<div class="text-center hover:text-cheerp">
+					<a
+						href="https://github.com/leaningtech/cheerp-meta?tab=License-1-ov-file#readme"
+					>
+						<p class="text-2xl font-bold">Apache 2.0</p>
+						<p>We are open source!</p>
+					</a>
+				</div>
+				<div class="text-center hover:text-cheerp">
+					<a href="https://github.com/leaningtech/cheerp-meta/stargazers">
+						<p class="text-2xl font-bold">1,000+</p>
+						<p>Stars on our GitHub repository</p>
+					</a>
+				</div>
+				<div class="text-center hover:text-cheerp">
+					<a href={DISCORD_URL}>
+						<p class="text-2xl font-bold">900+</p>
+						<p>Discord members</p>
+					</a>
+				</div>
+			</div>
+		</div>
+	</section>
+	<section
+		class="bg-gradient-to-b from-stone-900 to-[#474484] pb-48 text-center pb-28 md:px-8 md:pt-16 lg:px-32 lg:pt-32 xl:pt-42 xl:px-28 2xl:px-60 2xl:pt-60"
+	>
+		<p class="text-center text-bold text-5xl pb-20">
+			The enterprise-grade C/C++ compiler for the Web
+		</p>
+
+		<div class="px-16 grid sm:grid-cols-1 md:grid-cols-3 space-x-4">
+			<div class="col-span-2 text-left font-bold text-2xl py-4">
+				<p>
+					<span class="text-white">Do you need more?</span> Explore <span
+						class="text-cheerp"
+					>
+						Cheerp Enterprise</span
+					>
+				</p>
+				<ul class="list-inside list-disc font-normal text-wrap text-lg py-4">
+					<li>Technical support with tailored SLAs.</li>
+					<li>Feature prioritisation and sponsored development available.</li>
+					<li>
+						Access to our development team for compilation log reviews, video
+						calls and consulting.
+					</li>
+				</ul>
+				<LinkButton
+					type="primary-cheerp"
+					href="https://leaningtech.com/contact-us/?subject=Cheerp%20Enterprise%20Licence#mailus"
+					label="Contact us"
+				/>
+			</div>
+			<div class="col-span-1 border-solid border-l text-left py-4">
+				<p class="text-left text-2xl font-bold text-stone-300 pl-8">
+					<span class="text-white">Ready to build?</span> Get started today by browsing
+					our
+					<a href="/docs" class="text-cheerp hover:text-white">documentation</a>
+					and
+					<a href="/docs/tutorials" class="text-cheerp hover:text-white"
+						>tutorials</a
+					>
```

Comment:
I noticed that the documentation and tutorials are jump links, but they are not very obvious, because the ‘Cheerp Enterprise’ text on the left is the same style but not a link, so I thought maybe it might be better to put external link indicators next to them, here is a solution I found for your reference:

```suggestion
					<a
						href="/docs"
						class="text-cheerp hover:text-white inline-flex items-center"
						target="_blank"
						>documentation
						<svg
							class="w-4 h-4 ml-1"
							fill="currentColor"
							viewBox="0 0 20 20"
							xmlns="http://www.w3.org/2000/svg"
						>
							<path
								d="M11 3a1 1 0 100 2h2.586l-6.293 6.293a1 1 0 101.414 1.414L15 6.414V9a1 1 0 102 0V4a1 1 0 00-1-1h-5z"
							></path>
							<path
								d="M5 5a2 2 0 00-2 2v8a2 2 0 002 2h8a2 2 0 002-2v-3a1 1 0 10-2 0v3H5V7h3a1 1 0 000-2H5z"
							></path>
						</svg>
					</a>
```